.env

```
@@ -1,20 +1,35 @@
-# For test end ponint which depends on the login
+# ═══════════════════════════════════════════════════════════
+#  Authentication & Security
+# ═══════════════════════════════════════════════════════════
+NEXTAUTH_URL="http://localhost:3000"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)